### PR TITLE
fix service-worker in native automation

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

File: src/native-automation/request-pipeline/index.ts [hunk 1]
```diff
@@ -403,8 +404,9 @@ export default class NativeAutomationRequestPipeline extends NativeAutomationApi
         await this._client.Target.on('attachedToTarget', async event => {
             const isIFrame = event.targetInfo.type === TARGET_INFO_TYPE.iframe;
             const isWorker = event.targetInfo.type === TARGET_INFO_TYPE.worker;
+            const isServiceWorker = event.targetInfo.type === TARGET_INFO_TYPE.serviceWorker;
```
Comment (Aleksey28) [inline src/native-automation/request-pipeline/index.ts:407]: Align, please

File: test/functional/fixtures/regression/gh-8054/pages/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<script>
+    const registerServiceWorker = async () => {
+        if ("serviceWorker" in navigator) {
+            try {
+                const serviceWorkerUrl = '/fixtures/regression/gh-8054/pages/sw.js';
+
+                const registration = await navigator.serviceWorker.register(serviceWorkerUrl, {
+                    scope: "/fixtures/regression/gh-8054/pages/",
+                });
+                if (registration.installing) {
```
Comment (Aleksey28) [inline test/functional/fixtures/regression/gh-8054/pages/index.html:19]: Are you sure that we need these console logs?

**Review by Aleksey28 — CHANGES_REQUESTED** [review]
Great, please check my comments

File: test/functional/fixtures/regression/gh-8054/pages/index.html (hunk 2 above)
Comment (AlexKamaev) [reply test/functional/fixtures/regression/gh-8054/pages/index.html:19]: These logs does not affect the test execution. I can remove it.

**Pull request merged**